package.json

```diff
@@ -14,7 +14,7 @@
     "prepublishOnly": "npm run build",
     "start": "NODE_ENV=development BABEL_ENV=development webpack-dev-server --config=config/webpack.dev.config.js --hot --inline --progress",
     "debug-test": "node --inspect-brk node_modules/.bin/jest --runInBand --coverage",
-    "test": "jest --coverage --runInBand",
+    "test": "jest --coverage --maxWorkers=50%",
```

Comment:
this speeds up tests significantly and also makes use of half of the cores available which optimizes jest's use of resources